verifact/error_strings.py

```diff
@@ -1,4 +1,5 @@
 URL_FORMAT_INVALID = "The citation url is not a valid url. Please remember to include the scheme (eg. https://)"
 ANSWER_CHOICES_INVALID = "Answer must be True, False, or Neither"
 QUESTION_ID_INVALID = "Ensure this Answer is created in response to a valid Question!"
+QUESTION_ID_INVALID = "Ensure this Vote is created in response to a valid Answer!"
```

Comment:
Looks like you have a duplicate value, maybe add the related model to the variable name, i.e. `VOTE_QUESTION_ID_INVALID`. Or you could probably get away without specifying the related model and reusing as a single variable.

Reply:
typo - renaming this to `VOTE_ANSWER_ID_INVALID`. I think I'll rename line 3 to `ANSWER_QUESTION_ID_INVALID` as well to keep it consistent and a bit more specific